crates/ruff_python_ast/src/strings.rs

```diff
@@ -1,26 +1,33 @@
-use ruff_python_stdlib::str::{
-    SINGLE_QUOTE_PREFIXES, SINGLE_QUOTE_SUFFIXES, TRIPLE_QUOTE_PREFIXES, TRIPLE_QUOTE_SUFFIXES,
-};
-
 /// Strip the leading and trailing quotes from a docstring.
 pub fn raw_contents(contents: &str) -> &str {
-    for pattern in TRIPLE_QUOTE_PREFIXES {
+    for pattern in ruff_python_stdlib::str::TRIPLE_QUOTE_PREFIXES
```

Comment:
Nit: When do you prefer full qualified paths vs importing them at the top?

Reply:
(Moved to #3396, but I'll answer here that in this case, I'm using fully-qualified because `ruff_python_stdlib::str` and `ruff_python_stdlib::bytes` have identical member names.)